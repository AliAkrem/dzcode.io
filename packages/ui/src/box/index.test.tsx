import { render } from "@testing-library/react";

import { Box } from ".";

it("should render Box component", () => {
  const { container } = render(<Box />);
  expect(container).toMatchSnapshot();
});
