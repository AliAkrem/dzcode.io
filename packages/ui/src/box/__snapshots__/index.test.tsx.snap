// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Box component 1`] = `
<div>
  <div
    class="MuiBox-root css-0"
  />
</div>
`;
